pkg/store/container/metadata.go

```diff
@@ -56,6 +56,8 @@ type Metadata struct {
 	Config *runtime.ContainerConfig
 	// ImageRef is the reference of image used by the container.
 	ImageRef string
+	// StopSignal is the signal to send to stop the container.
+	StopSignal string
```

Comment:
out of habit can we add new stuff to the end of the struct?

Reply:
Good point, will do.

Reply:
Done